app/services/sequelizeErrorParser.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (errors) => {
+  if (errors.name !== 'SequelizeValidationError') {
+    return null;
+  }
+
+  const err = {};
+  errors.errors.forEach(error => {
```

Comment:
Please try this one https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce